index.html

```diff
@@ -0,0 +1,580 @@
+                        </svg></a></li>
+                
+                    <li><a class="social-icons" href="#"><svg class="mail" xmlns="http://www.w3.org/2000/svg" height="24px" viewbox="0 0 24 24"
+                            width="24px" fill="#000000">
+                            <path d="M0 0h24v24H0V0z" fill="none" />
+                            <path
+                                d="M22 6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6zm-2 0l-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z" />
+                        </svg></a></li>
+                
+                    <li><a class="social-icons" href="#"><svg class="share" xmlns="http://www.w3.org/2000/svg" height="24px" viewbox="0 0 24 24"
+                            width="24px" fill="#000000">
+                            <path d="M0 0h24v24H0V0z" fill="none" />
+                            <path d="M10 9V5l-7 7 7 7v-4.1c5 0 8.5 1.6 11 5.1-1-5-4-10-11-11z" />
+                        </svg></a></li>
+                
+                    <li><a class="social-icons" href="#"><svg class="comment-box" xmlns="http://www.w3.org/2000/svg" height="24px" viewbox="0 0 24 24"
+                            width="24px" fill="#000000">
+                            <path d="M0 0h24v24H0V0z" fill="none" />
+                            <path
+                                d="M20 17.17L18.83 16H4V4h16v13.17zM20 2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4V4c0-1.1-.9-2-2-2z" />
+                        </svg>
+                    </a></li>
+            </ul>
+        </div>
+
+
+        <figure class="main-p">
+            <img class="alan-guth" src="images/18cosmos-jumbo.jpg" alt="">
+            <figcaption>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which
+                explains
+                how the <br> universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+                billion
+                years
+                ago. <cite>Rick <br> Friedman for The New York Times</cite>
+            </figcaption>
+        </figure>
```

Comment:
- [ ] Please I'd suggest position this segment properly, it doesn't look bad on desktop (not that it is all good) but when in mobile version (for chrome - use `ctrl+shift+I` to inspect), it looks distorted and very narrow.